modules/tests/commentCount.test.js

```diff
@@ -0,0 +1,49 @@
+import commentCount from '../commentCount.js';
+import displayComments from '../displayComments.js';
+
+const comments = [
+  {
+    creation_date: '2023-02-08',
+    username: 'aubin',
+    comment: 'Nice one',
+  },
+  {
+    creation_date: '2023-02-08',
+    username: 'John',
+    comment: 'Good',
+  },
+  {
+    creation_date: '2023-02-08',
+    username: 'Marc',
+    comment: 'Great',
+  },
+
+];
+describe('TodoApp', () => {
+  test('Number of comments In the DOM before API fetch', () => {
+   document.body.innerHTML = `
+   <div class="comment-container">
+   <h3 id="comment-title">Comments </h3>
+   <div id="comment-list">
+
+   </div>
+  </div`;
+    const section = document.getElementById('comment-list');
+    expect(commentCount(section)).toHaveLength(0);
+  });
+
+  test('Number of comments displayed in the DOM after a successful API call', () => {
+    document.body.innerHTML = `
+ 
+    <div class="comment-container">
+    <h3 id="comment-title">Comments </h3>
+    <div id="comment-list">
+
+    </div>
+   </div>`;
+   const section = document.getElementById('comment-list');
+
+    displayComments(comments, section);
+    expect(commentCount(section)).toHaveLength(3);
+  });
+});
```

Comment:
As shown in the image below, the tests for the commentCounter are failing. Kindly check and ensure that they all pass. :wink: 

![Screenshot from 2023-02-10 09-10-05](https://user-images.githubusercontent.com/65233860/218050855-9ad44de4-30f4-4a0d-93e6-7165515a373f.png)